data/mods/Aftershock/items/gun/25mm.json

```
@@ -0,0 +1,48 @@
+[
+  {
+    "id": "afs_gibs_shotgun",
+    "looks_like": "m79",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "Gibson S86" },
+    "description": "The Gibson S86 is an extremely powerful 5-gauge combat shotgun manufactured by Gibson Armory, designed to take care of heavily armored opponents like power armor users.  However due to its recoil, the gun is only usable if its either mounted, or wielded by a person with sufficent strength like a power armor user.\n\nDue to its size and build, it feels more like a grenade launcher than an actual shotgun, and can actually chamber light grenade ammo.",
+    "weight": "2700 g",
+    "volume": "2800 ml",
+    "longest_side": "736 mm",
+    "price": 550000,
+    "price_postapoc": 50000,
+    "to_hit": -3,
+    "bashing": 10,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "brown",
+    "min_strength": 18,
+    "ranged_damage": { "damage_type": "bullet", "amount": 10 },
+    "skill": "launcher",
+    "modes": [ [ "DEFAULT", "auto", 6 ] ],
```

This burst is probably too long. Havent play tested it but wouldnt the last 3 tot 2 shots just completely miss almost allways?

---

Correct yeah; most of em miss

The idea was it is very inaccurate, however if this is a problem I can lower it.

---

Dont think balancing a heavy weapon around most shots missing is exactly right.  A precise long burst is fine because it easily kills strong enemies and wastes a lot of  ammo overkilling weak foes. But a non precise burst isnt really good at anything but wasting your resources, and would likely mean that the gun will go unused.

A high miss percentage is fine if you dont have skills and randomly pick up the gun while having the mininum requirements. But someone with the adequate strenght boosts + skills should probably be able to land the whole burst against large oponents.

---

```suggestion
    "modes": [ [ "DEFAULT", "multi", 6 ] ],
```
This might allow the 6 bullets to hit without needing many overreaching changes. It would have to be tested however.

---

Can do; I'll try testing it out later

---

I ended up lowering it down to 4 and setting it to DOUBLE, as no matter how I fiddle with it; every shot missed until I lowered it down.